Controller/Redirect/Process.php

```diff
@@ -718,6 +723,7 @@ public function secondChance(){
             'store_id' => $order->getStoreId(),
             'created_at' => $this->dateTime->gmtDate(),
         ]);
+        $this->quoteRecreate->duplicate($order);
```

Comment:
what is the workflow here - would like to understand the context, why this has to be part of the constructor...

Reply:
Customer after restore session use same quote as in order, so in case he modify quote, it will be modify in second chance to, so in this case added quote duplicate